designer_v2/lib/features/monitor/participant_details_view.dart

```diff
@@ -0,0 +1,327 @@
+          EmptyBody(
+            icon: Icons.hourglass_empty,
+            title: tr.participant_details_progress_empty_title,
+            description: "",
+          ),
+        const SizedBox(height: 16.0),
+        _buildColorLegend(),
+      ],
+    );
+  }
+
+  Widget _buildParticipantInfo(String languageCode) {
+    return SelectionArea(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          _buildInfoRow(
+            tr.monitoring_table_column_participant_id,
+            monitorItem.participantId,
+          ),
+          _buildInfoRow(
+            tr.monitoring_table_column_invite_code,
+            monitorItem.inviteCode ?? 'N/A',
+          ),
+          _buildInfoRow(
+            tr.monitoring_table_column_enrolled,
+            monitorItem.startedAt
+                .toLocalizedString(locale: languageCode, showTime: false),
+          ),
+          _buildInfoRow(
+            tr.monitoring_table_column_last_activity,
+            monitorItem.lastActivityAt.toLocalizedString(locale: languageCode),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildInfoRow(String label, String value) {
+    return Row(
+      children: [
+        Text(
+          '$label: ',
+          style: const TextStyle(fontWeight: FontWeight.bold),
+        ),
+        Text(value),
+      ],
+    );
+  }
+
+  Widget _buildPerDayStatus() {
+    final int totalCompletedDays = monitorItem.missedTasksPerDay.length;
+
+    final phases = <StudyPhase>[];
+
+    if (studySchedule.includeBaseline) {
+      phases.add(
+        StudyPhase(
+          name: 'Baseline',
+          missedTasksPerDay: monitorItem.missedTasksPerDay.sublist(
+            0,
+            totalCompletedDays > studySchedule.baselineLength
+                ? studySchedule.baselineLength
+                : totalCompletedDays,
+          ),
+        ),
+      );
+    }
+
+    final String sequence = studySchedule.nameOfSequence;
+
+    for (int i = 0; i < studySchedule.numberOfCycles * 2; i++) {
+      final int phaseDuration = studySchedule.phaseDuration;
+      final bool includeBaseline = studySchedule.includeBaseline;
+      final int baselineAdjustmentStart =
+          includeBaseline ? (i + 1) * phaseDuration : i * phaseDuration;
+      final int baselineAdjustmentEnd =
+          includeBaseline ? (i + 2) * phaseDuration : (i + 1) * phaseDuration;
+
+      if (baselineAdjustmentStart >= totalCompletedDays) break;
+
+      final int start = totalCompletedDays > baselineAdjustmentStart
+          ? baselineAdjustmentStart
+          : totalCompletedDays;
+
+      final int end = totalCompletedDays > baselineAdjustmentEnd
+          ? baselineAdjustmentEnd
+          : totalCompletedDays;
+
+      var sequenceIndex = i;
+      if (sequenceIndex > 3) sequenceIndex = i % 4;
+
+      final String interventionName = sequence[sequenceIndex] == "A"
+          ? interventions[0].name!
+          : interventions[1].name!;
```

Comment:
And for the sake of clarity, how about prefixing the intervention name with their letter, as it is done on the intervention page:
<img width="770" alt="image" src="https://github.com/hpi-studyu/studyu/assets/104295301/b631a13f-eb59-4132-a53a-bd0dc967e25f">

Reply:
<img width="302" alt="image" src="https://github.com/hpi-studyu/studyu/assets/104295301/1b22bdff-f0d6-48e8-94d1-1e2980c3b642">

We probably do not need to show the name of the intervention again in the tooltip, since it is already in the heading.